src/Template/Contributions/activity_timeline.ctp

```diff
@@ -64,7 +60,10 @@ if (!empty($stats)) {
 
 <div id="main_content">
     <div class="section md-whiteframe-1dp">
-    <h2><?php echo __('Activity timeline') . ' - ' . $selectedMonth; ?></h2>
+    <h2><?php
+        echo __('Activity timeline') . ' &#8212; ' .
+            format(__('{month} {year}'), array('month' => $monthName, 'year' => $year));
```

Comment:
Sorry, I think I wasn’t clear. The whole string should be translatable so that translators can totally and easily rearrange the whole title. Your code forces translators to use the dash character and space and to use the two strings in that order. However, some languages don’t use the dash character or even don’t use the space character. And you can never be sure about the word order. So it should be like this:
```php
echo format(__('Activity timeline — {month} {year}'), array('month' => $monthName, 'year' => $year));
```

Reply:
About the dash character, as far as I know [we use hyphen everywhere else in titles](https://github.com/Tatoeba/tatoeba2/blob/d39ecb565a894a661c137302bedbc2eb527e16a0/src/View/Helper/PagesHelper.php#L28), so for consistency it’s probably better to stick with the hyphen for now.